dub.sdl

```diff
@@ -49,3 +49,105 @@ subPackage {
 
   dependency "dmd:lexer" version="*"
 }
+
+subPackage {
+  name "frontend"
+  targetType "library"
+  stringImportPaths "res"
+  versions "NoBackend"
+  versions "GC"
+  versions "NoMain"
+  sourcePaths
+
+  sourceFiles \
```

Comment:
One way is to move every file that is part of the Dub package to its own directory, then the whole directory can be included with `sourcePaths`.

Reply:
DUB also supports `excludedSourceFiles` for removing files from a `sourcePaths` set

Reply:
Give it a try. Seems to be simple enough, though it would have been easier with a better layout of DMD (see the exclude file globs) and I don't know how fast the globbing is implemented in DUB (though it's a handful of files)...

If you want to cleanup the house, some ideas:
- move everything related to the IR -> `dmd.ir`
- move everything related to reading & writing object files `src/dmd/{scan,lib}{elfmach,mscoff,omf}.d` -> `dmd.obj`